modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-log-forwarding.adoc
+
+[id="cluster-logging-log-forwarding-about_{context}"]
+= Understanding cluster log forwarding
+
+The {product-title} cluster log forwarding feature uses a combination of _outputs_ and _pipelines_ defined in the Log Forwarding Custom Resource to send logs to specific endpoints inside and outside of your {product-title} cluster. 
+
+An output is the destination for log data and a pipeline defines simple routing for one source to one or more outputs. 
+
+An output can be either:
+
+* `elasticsearch` to forward logs to an external Elasticsearch v5.x cluster, specified by server name or FQDN.
```

Comment:
As described in https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-log-forwarding.md#proposal, seems we support 6.x elasticsearch, but I didn't test the 5.x ES, not sure if it's supported or not. @jcantrill could you please help to confirm?

Reply:
This is a subtle mistake.  LF is being released before we deliver ES6 so technically we support writing to ES5 and have not fully tested ES6.  I leave that to you if you tested it, otherwise we support ES5

Reply:
In my testing, I use `docker.elastic.co/elasticsearch/elasticsearch:6.8.4`